stix_shifter/stix_translation/src/modules/bigfix/json/to_stix_map.json

```diff
@@ -31,14 +31,43 @@
         "key": "file.hashes.MD5",
         "object": "file"
     },
+    "file_size": {
+        "key" : "file.size",
+        "object": "file",
+        "transformer": "ToInteger"
+    },
     "process_name": {
         "key": "process.name",
         "object": "process"
     },
     "process_id": {
         "key": "process.pid",
-        "object": "process"
+        "object": "process",
+        "transformer": "ToInteger"
     },
+   "process_ppid": [
+    {
+      "key": "process.pid",
+      "object": "parent_process",
+      "transformer": "ToInteger"
+    },
+    {
+      "key": "process.parent_ref",
+      "object": "process",
+      "references": "parent_process"
+    }
+    ],
+    "process_user": [
+      {
+        "key":"user-account.user_id",
+        "object": "user"
+      },
+      {
+        "key": "process.creator_user_ref",
+        "object": "process",
+        "references": "user"
+      }
+    ],
     "start_time": [
         {
             "key": "created",
```

Comment:
we are removing the `created` field from the mapping file and setting the `created` and `modified` timestamp as the current time. we are merging an update shortly in master branch. please pull it down the latest change when it is merged.